snips_nlu/tests/test_probabilistic_intent_parser.py

```diff
@@ -42,11 +43,13 @@ def test_should_parse(self):
 utterances:
   - foz for [slot3:entity3](baz)""")
         dataset = Dataset.from_yaml_files("en", [dataset_stream]).json
-        classifier_config = LogRegIntentClassifierConfig(random_seed=42)
-        slot_filler_config = CRFSlotFillerConfig(random_seed=42)
+        classifier_config = LogRegIntentClassifierConfig()
+        slot_filler_config = CRFSlotFillerConfig()
         parser_config = ProbabilisticIntentParserConfig(
             classifier_config, slot_filler_config)
```

Comment:
`classifier_config`, `slot_filler_config ` and `parser_config` are not needed here